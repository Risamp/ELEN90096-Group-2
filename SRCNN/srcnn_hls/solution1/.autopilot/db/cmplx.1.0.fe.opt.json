{
  "0": {
    "basicblocks": 1,
    "calls": "3 1 4",
    "file": "srcnn.cpp",
    "fileloc": 4,
    "instructions": 4,
    "name": "srcnn",
    "total_instructions": 437
  },
  "1": {
    "basicblocks": 39,
    "calls": "2 2",
    "file": "conv2.cpp",
    "fileloc": 8,
    "instructions": 122,
    "loads": {
      "L": 6
    },
    "name": "conv2",
    "stores": {
      "L": 3
    },
    "total_instructions": 140
  },
  "2": {
    "basicblocks": 6,
    "calls": "",
    "file": "util.cpp",
    "fileloc": 82,
    "instructions": 9,
    "name": "clamp",
    "total_instructions": 9
  },
  "3": {
    "basicblocks": 51,
    "calls": "2 2",
    "file": "conv1.cpp",
    "fileloc": 9,
    "instructions": 152,
    "loads": {
      "L": 6
    },
    "name": "conv1",
    "stores": {
      "L": 3
    },
    "total_instructions": 170
  },
  "4": {
    "basicblocks": 37,
    "calls": "2 2",
    "file": "conv3.cpp",
    "fileloc": 8,
    "instructions": 105,
    "loads": {
      "L": 5
    },
    "name": "conv3",
    "stores": {
      "L": 2
    },
    "total_instructions": 123
  }
}