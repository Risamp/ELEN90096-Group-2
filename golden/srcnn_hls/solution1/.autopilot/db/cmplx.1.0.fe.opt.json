{
  "0": {
    "basicblocks": 1,
    "calls": "2 4 1",
    "file": "srcnn.cpp",
    "fileloc": 4,
    "instructions": 4,
    "name": "srcnn",
    "total_instructions": 407
  },
  "1": {
    "basicblocks": 37,
    "calls": "3 3",
    "file": "conv3.cpp",
    "fileloc": 8,
    "instructions": 105,
    "loads": {
      "L": 5
    },
    "name": "conv3",
    "stores": {
      "L": 2
    },
    "total_instructions": 123
  },
  "2": {
    "basicblocks": 39,
    "calls": "3 3",
    "file": "conv1.cpp",
    "fileloc": 9,
    "instructions": 122,
    "loads": {
      "L": 6
    },
    "name": "conv1",
    "stores": {
      "L": 3
    },
    "total_instructions": 140
  },
  "3": {
    "basicblocks": 6,
    "calls": "",
    "file": "util.cpp",
    "fileloc": 82,
    "instructions": 9,
    "name": "clamp",
    "total_instructions": 9
  },
  "4": {
    "basicblocks": 39,
    "calls": "3 3",
    "file": "conv2.cpp",
    "fileloc": 8,
    "instructions": 122,
    "loads": {
      "L": 6
    },
    "name": "conv2",
    "stores": {
      "L": 3
    },
    "total_instructions": 140
  }
}